comictalker/talkers/comicvine.py

```diff
@@ -772,6 +763,7 @@ def fetch_issue_data_by_issue_id(self, issue_id: int) -> GenericMetadata:
             self.settings_options["use_series_start_as_volume"]["value"],
         )
 
+    # TODO Why is this required? Was this to get around ii and empty URL?
```

Comment:
Well here's the logo: https://comicvine.gamespot.com/a/bundles/comicvinesite/images/logo.png
![comicvine.png]

[comicvine.png]: https://comicvine.gamespot.com/a/bundles/comicvinesite/images/logo.png

Reply:
You must search better than me! Still, any reason to keep the function? No point comparing that logo to a cover?

Reply:
If you can find me a comic that is missing it's cover to test with we can remove it

Reply:
I had to sign up again to CV and see where you got the logo now! :) Anyway... I had a short search and couldn't find one so looked at adding volumes/issues. When it comes to adding a volume an image is required. For an issue it is not (going by the form). They may refuse issues without an image but it looks like an issue without an image is a possibility.

The function also seems redundent because the CV data is now formatted to ComicVolume/Issue so if you want to keep it, putting in the format function makes more sense?

Reply:
Yes you already take care of this in your format* functions